InformationScripting/src/query_prompt/QueryBuilder.cpp

```diff
@@ -89,7 +95,14 @@ Query* QueryBuilder::visitOperator(QueryBuilder* self, OperatorQueryNode* op)
 	else if (leftComposite)
 	{
 		// union
-		connectQueriesWith(composite, leftComposite, new UnionOperator(), right);
+		Query* unionQuery = nullptr;
+		if (op->op() == OperatorQueryNode::OperatorTypes::Pipe)
+			unionQuery = composite->addQuery(std::unique_ptr<Query>(new UnionOperator()));
```

Comment:
Now that we've decided to use | for pipes, unions and splits, we should probably rename this operator to `PipeOperator`. What do you think?